services/commissioning.service

```diff
@@ -0,0 +1,19 @@
+[Unit]
+Description=Rovercode Commissioning Service
+After=syslog.target docker.service
+Requires=docker.service
+
+[Service]
+TimeoutStartSec=0
+Restart=always
+User=root
+Group=root
+ExecStartPre=-/usr/bin/docker stop rovercode-commissioning
+ExecStartPre=-mkdir -p /mnt/usb-stick && mount /dev/sda1 /mnt/usb-stick
```

Comment:
Not super robust, but probably enough to get us by in our controlled environment.